icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -1131,30 +1131,30 @@ class U_I18N_API Scale : public UMemory {
 
 namespace impl {
 
-// Do not enclose entire Usage with #ifndef U_HIDE_INTERNAL_API, needed for a protected field
+// Do not enclose entire StringProp with #ifndef U_HIDE_INTERNAL_API, needed for a protected field
 /**
  * Manages NumberFormatterSettings::usage()'s char* instance on the heap.
  * @internal
  */
-class U_I18N_API Usage : public UMemory {
+class U_I18N_API StringProp : public UMemory {
```

Comment:
I think also `StringProp` is better to be moved to Utilities

Reply:
This is NumberFormatter specific. E.g. friend classes:

    friend class NumberFormatterImpl;

It's a class for a MacroProps property, just like SymbolsWrapper, Grouper, and Padder, so I believe it belongs where those classes are found.